### DISCUSSION TITLE
No Connection to Plex after Setup the new "Test Server"

### DISCUSSION THREAD

**Discussion by simons25** [lostb1t/replex#102] [Q&A]
Hello,

thx for this great tool. However i have problems to get a connection. i used your test server and as indicated with base32 my Plex ip in front.
if i now go to the url provided, replex works. it shows the custom rows.
However, any connection is no longer available as soon as I enter the url in the custom domain field and disable remote access?

**Comment by lostb1t** [comment]
make sure secure connections is set to prefered. And clear your cache on your clients. It should start working omonyour client after that. (or on app.plex.tv)

**Reply by simons25** [reply]
Secure Connections are set to preferred and the server is completely gone after disabling remote access. Clearing cache change nothing...any other idea? ;)

**Reply by lostb1t** [reply]
gone from where?

what happens if you goto app.plex.tv?

**Reply by simons25** [reply]
I mean the server is completely not reachable from outside...only direkt on server on "localhost:32400".
If I go to app.plex.tv it shows this:


![Bildschirmfoto 2023-11-08 um 11 30 45](https://github.com/lostb1t/replex/assets/150127011/ec50231e-a674-42ba-8f9f-747b8b3d7e68)
![Bildschirmfoto 2023-11-08 um 11 30 37](https://github.com/lostb1t/replex/assets/150127011/0809e11b-e78a-4a95-b127-f8836434eca2)


"Check your network connection and make sure that all media servers are online."
